### ISSUE TITLE
Theme resets upon startup

### ISSUE THREAD

**Issue by Kenst1092** [namazso/SecureUxTheme#128]
I am currently stumped with this problem. Upon startup, the theme appears to be reset and I have to re-run SecureUXTheme and reapply my theme to get my chosen custom theme to work.

I'm on Windows 11 22621.963 RTM. **LogonUI is patched**. TPM 2.0 enabled, Secure Boot disabled. Running SecureUXTheme 2.1.2. I'm reaching out to you people to help me out, and I will provide more information as requested.

Currently trying out a potential mitigation with rehooking into LogonUI, as Windows did go through two cumulative updates that increased the build version number since the last time I patched via SecureUXTheme.

P.S. Doesn't work, lol. Still resets upon startup

**Comment by Crocodile73** [comment]
yeah i started getting this as well. No idea why. What i found is, if you apply a theme and then logout and back in, whenever you then shutdown or restart, it will apply. Simply changing themes and restarting or shutting down, loads the previous theme. Windows x64 22621.2213.

**Comment by nopeless** [comment]
@namazso Any way I could help?
